src/ozone/ozone.py

```diff
@@ -315,6 +315,8 @@ def get_coordinate_air(
                 append the data to.
             params (List[str], optional): A list of parameters to get data for.
             Gets all parameters by default.
+								Choose from the following values:
+                    [ "aqi", "pm25", "pm10", "o3", "co", "no2", "so2", "dew", "h", "p", 
```

Comment:
@tariksouabny  thanks for addressing the requests,
the new changes you made however seem to be missing few parameters which need to be added  
```suggestion
								Choose from the following values:
                    [ "aqi", "pm25", "pm10", "o3", "co", "no2", "so2", "dew", "h", "p", "t", "w", "wg" ]
```